src/_includes/layouts/post.njk

```diff
@@ -1,8 +1,14 @@
 ---
 layout: layouts/base.njk
 type: post
+eleventyComputed:
```

Comment:
Not quite sure why this direction was chosen over the linked blog post, with a new collection? I feel this approach is a bit too much magic.

Reply:
I created the new collection, this is just setting the permalink to false so it doesn't generate the pages of future posts in production, but keep them visible in development